pynucastro/networks/tests/_python_reference/network.py

```diff
@@ -63,6 +69,34 @@ def __init__(self):
         self.he4_c12__o16 = np.nan
         self.n__p__weak__wc12 = np.nan
         self.he4_he4_he4__c12 = np.nan
+        self.na23__ne23 = np.nan
+        self.ne23__na23 = np.nan
+
+# load data for na23 --> ne23
+na23__ne23_table_path = '/home/runner/work/pynucastro/pynucastro/pynucastro/library/tabular/23Na-23Ne_electroncapture.dat'
```

Comment:
I wonder if hardcoding the path here is safe -- it could change in the future.  Do we have a mechanism to get the path otherwise?

Reply:
I was just thinking about that too. Looks like we usually look in a few places (workdir, `pynucastro/library`, and `pynucastro/library/tabular`). So we can look for the file in those places instead of declaring a full path.